### ISSUE TITLE
Outdated packages when installing package

### ISSUE THREAD

**Issue by LvffY** [cloud-carbon-footprint/ccf-backstage-plugin#10]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/cloud-carbon-footprint/ccf-backstage-plugin/blob/trunk/CONTRIBUTING.md) for this project.
- [X] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

- **Operating System:** Debian 10.1
- **Browser:** N/A

### Expected Behavior

Cloud carbon footprint plugin should be installed without any warnings.

### Actual Behavior

- When running `yarn --cwd packages/backend add @cloud-carbon-footprint/backstage-plugin-backend` 

![image](https://user-images.githubusercontent.com/37327656/174429930-7b904600-1385-4db2-9575-0953c2b12c9e.png)

- When running `yarn --cwd packages/app add @cloud-carbon-footprint/backstage-plugin-frontend`
![image](https://user-images.githubusercontent.com/37327656/174429945-c7394e24-b68c-4a88-898e-f9414192f891.png)

### To Reproduce

Just install the cloud carbon footprint plugin

### Additional Information

© 2022 Thoughtworks, Inc.

**Comment by 4upz** [comment]
Hi @LvffY, thanks for bringing this to our attention! Unfortunately, this is expected behavior for now. Both of the screenshots in question refer to the Azure Node SDK (the`arm-consumption` package for interacting with the Consumption Management API) and React packages (`react-scriptes` sub-dependency). Both of these packages' latest releases have breaking changes that we're waiting on being fixed before upgrading our app packages to their latest version. If any of these packages are causing an error, feel free to open a new issue with the specifics in the [CCF Repo](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint).

For extra details, the compatibility of these dependency issues are covered here:

- React: https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/issues/813
- Azure: https://github.com/Azure/azure-sdk-for-js/issues/20026
